src/Prima/Pyxis/Modal.elm

```diff
@@ -0,0 +1,814 @@
+module Prima.Pyxis.Modal exposing (Config, hide, large, medium, render, show, small, toggle, withAttribute, withBodyAttribute, withBodyClass, withBodyClassList, withBodyContent, withBodyStyle, withClass, withClassList, withCloseOnOverlayClick, withFooterAttribute, withFooterClass, withFooterClassList, withFooterContent, withFooterStyle, withHeaderAttribute, withHeaderClass, withHeaderClassList, withHeaderContent, withHeaderContentOnly, withHeaderStyle, withHeaderTitle, withHeaderTitleOnly, withId, withOverlayAttribute, withOverlayClass, withOverlayClassList, withStyle, withTitleAttribute)
+
+import Html exposing (Html)
+import Html.Attributes as HtmlAttributes
+import Prima.Pyxis.Helpers as H
+import Prima.Pyxis.Shared.InterceptedEvents as InterceptedEvents
+import Prima.Pyxis.Shared.Interceptor as Interceptor
+
+
+{-| Represent the configuration of a `Container`.
+This type can't be created directly and is Opaque.
+You must use Configuration constructors instead
+-}
+type Config msg
```

Comment:
Se non sbaglio negli altri moduli abbiamo chiamato questo tipo `NomeComponente` e record interno `NomeComponenteConfig`. Manterrei quella nomenclatura, quindi `Modal` e `ModalConfig`

Reply:
Mi pareva avessimo detto proprio il contrario, avevo fatto così in Container e l'ho cambiato ...